spec/controllers/users_controller_spec.rb

```diff
@@ -82,5 +97,55 @@
         expect(assigns(:user)).to eq(user)
       end
     end
+    describe 'GET #edit' do
+      it 'shows edit page' do
+        user = User.create! valid_attributes
+        get :edit, { id: user.to_param }, valid_session
+        expect(assigns(:user)).to eq(user)
+      end
+    end
+    describe 'PUT #update' do
+      context 'with valid params' do
+        let(:new_attributes) {
+          {
+            email: 'new_attributes@example.com',
+            password: 'aabbccdd',
+            role: 'user'
+          }
+        }
+
+        it 'updates the requested user' do
```

Comment:
Can't you just declare in a let block & re-use the user instance across the examples below?